lunes_cms/cms/models/document.py

```diff
@@ -31,6 +31,12 @@ class Document(models.Model):
         verbose_name=_("word type"),
     )
     word = models.CharField(max_length=255, verbose_name=_("word"))
+    grammatical_gender = models.IntegerField(
+        choices=Static.grammatical_genders,
+        verbose_name=_("grammatical gender"),
```

Comment:
```suggestion
        verbose_name=_("Grammatical gender"),
```
Although it's auto-capitalized by our template, I think we should capitalize it ourselves too